### PR TITLE
Persistence fix: Load DNA from source chain

### PR DESCRIPTION
**Pull request by lucksus**
For holochain_load() to really work we need to construct a state with everything a running Instance needs. And that means: the DNA.

* retrieving the DNA from the source chain on load
* putting it into the state and making sure that state is used in the newly created instance

### CODE REVIEW

File: dna/src/lib.rs [hunk 1]
```diff
@@ -264,7 +264,7 @@ impl ToEntry for Dna {
     }
 
     fn from_entry(entry: &Entry) -> Self {
-        return Dna::from_json_str(&entry.content()).expect("entry is not a valid Dna Entry");
+        return Dna::from_json_str(&entry.value()).expect("entry is not a valid Dna Entry");
```
Comment (lucksus) [inline dna/src/lib.rs]: THIS was a mean bug!

Comment (sphinxc0re) [reply dna/src/lib.rs]: Please remove the `return`

Comment (lucksus) [reply dna/src/lib.rs]: jeez

**Comment by lucksus** [comment]
Pushed removal of return, @sphinxc0re. Please merge at will.
(and just go ahead push such little changes in the future generally)

**Pull request merged**